src/components/Capture/capture.js

```diff
@@ -123,7 +118,7 @@ class Capture extends Component {
     else {
       postToServer(this.createJSONPayload(payload), serverUrl, token, (response) => {
         this.handleMessages(response)
```

Comment:
you can just pass `this.handleMesssages, this.onServerError`. I would also rename `handleMessages(message)` to `onServerResponse(response)`.

Otherwise looks good!